mail_client/api/mail.py

```diff
@@ -354,3 +357,17 @@ def get_mail_contacts(txt=None) -> list:
 			contact.update(details)
 
 	return contacts
+
+
+@frappe.whitelist()
+def get_default_outgoing() -> str:
+	"""Returns default outgoing mailbox."""
+
+	return frappe.get_all(
+		"Mailbox",
+		{
+			"user": frappe.session.user,
+			"is_default": 1,
+		},
+		pluck="name",
+	)[0]
```

Comment:
The result can be an empty list. Administrator user don't have a default mailbox.